tests/test_group.rs

```diff
@@ -159,109 +212,156 @@ fn basic_group_setup() {
     };
 }
 
-/*
 #[test]
 fn group_operations() {
```

Comment:
What operations?
This needs a description of what's exactly happening/tested in here.

Reply:
🤔

Reply:
That was added, but I think you are looking at the wrong file